api/base/exceptions.py

```diff
@@ -36,11 +43,30 @@ def json_api_exception_handler(exc, context):
     return response
 
 
+class JSONAPIException(APIException):
+    """Inherits from the base DRF API exception and adds extra metadata to support JSONAPI error objects
+
+    :param str detail: a human-readable explanation specific to this occurrence of the problem
+    param dict source: an object containing references to the source of the error, optionally including
```

Comment:
Typos here, and the Sphinx markup is invalid. It should be enough to link to the spec, as in:

```
:param dict source: A dictionary containing references to the source of the error.
    See http://jsonapi.org/format/#error-objects.
    Example: ``source={'pointer': '/data/attributes/title'}``
```